CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 #### 0.10.0
 
-- Allows to declare a schema inside the yaml file through modeline `# yaml-language-server: $schema=<urlOfTheSchema>`
+- Allows to declare a schema inside the yaml file through modeline `# yaml-language-server: $schema=<urlOfTheSchema>` [#280](https://github.com/redhat-developer/yaml-language-server/pull/280)
+- Insert empty string instead of 'null' for string array completion [#277](https://github.com/redhat-developer/yaml-language-server/pull/277)
+- Handle workspace folders better [#281](https://github.com/redhat-developer/yaml-language-server/pull/281)
```

Comment:
The changlog is intended to be read by the general user. The description should be more then the title of the bug or PR. I should be able to read this and know what it is and if it will effect me. If I have questions, I can open the link and read the details.

Reply:
Yeah that's partially my fault for the bad PR name. I think we should change it to something like:

```suggestion
- Handle workspace/workspaceFolders event for multi root workspaces [#281](https://github.com/redhat-developer/yaml-language-server/pull/281)
```